### PR TITLE
W-11464471-hosting-options

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/intro-platform-hosting.adoc [hunk 1]
```diff
@@ -114,14 +123,14 @@ You can manage standalone runtimes by using cloud control planes (US Cloud, EU C
...
-| Mule runtime engine | Y | Y | Y
-| Anypoint MQ | Y | N | N
-| Anypoint Object Store | Y | N | N
-| Anypoint DataGraph | Y | N | N
-| Connectors | Y | Y | Y
+| Anypoint Platform Component | CloudHub 2.0 | CloudHub | Runtime Fabric | Standalone
+| Mule runtime engine | Y | Y | Y | Y
+| Anypoint MQ | N | Y | N | N
+| Anypoint Object Store | Y | Y | N | N
+| Anypoint DataGraph | Y | Y | N | N
```
Comment (sarathecoubian) [inline modules/ROOT/pages/intro-platform-hosting.adoc]:
```suggestion
| Anypoint DataGraph | N | Y | N | N
```

File: modules/ROOT/pages/intro-platform-hosting.adoc [hunk 2]
```diff
@@ -86,13 +87,21 @@ Not all Anypoint Platform components are supported in all cloud environments. Th
 
 MuleSoft provides three different options for hosting the Anypoint Platform runtime plane:
 
+* CloudHub 2.0
++
+CloudHub 2.0 is MuleSoft's cloud-based environment for hosting the Mule runtime and related services. CloudHub 2.0 enables you to deploy an API or a Mule application into a container-based platform that's managed by MuleSoft. You can deploy to MuleSoft's shared spaces or your own configurable private spaces for more control over what can interact with your applications.
```
Comment (vli63127) [inline modules/ROOT/pages/intro-platform-hosting.adoc]:
```suggestion
CloudHub 2.0 is MuleSoft's new default cloud-hosted environment for hosting the Mule runtime and related services. CloudHub 2.0 enables you to deploy an API or a Mule application into a container-based platform that's managed by MuleSoft. You can deploy to MuleSoft's shared spaces or your own configurable single-tenant private spaces for more control over what can interact with your applications.
```

File: modules/ROOT/pages/intro-platform-hosting.adoc [hunk 3]
```diff
@@ -86,13 +87,21 @@ Not all Anypoint Platform components are supported in all cloud environments. Th
...
++
+CloudHub 2.0 also provides high availability, ingress load balancers, AWS role-based access controls, and endpoint configuration. See xref:cloudhub-2::index.adoc[CloudHub 2.0] for more information.
++
+CloudHub 2.0 can only be managed only by the version of the cloud control plane that exists in the same environment (US cloud or EU cloud).
+
 * CloudHub
 +
-CloudHub is MuleSoft's cloud-based environment for hosting the Mule runtime server and related services. CloudHub enables you to deploy an API or a Mule application on a platform that’s managed by MuleSoft.
+CloudHub is MuleSoft's original cloud-based environment for hosting the Mule runtime server and related services. CloudHub enables you to deploy an API or a Mule application on a platform that’s managed by MuleSoft.
```
Comment (vli63127) [inline modules/ROOT/pages/intro-platform-hosting.adoc]:
```suggestion
CloudHub is MuleSoft's original cloud-hosted environment for hosting the Mule runtime and related services. CloudHub enables you to deploy an API or a Mule application on a VM-based platform that’s managed by MuleSoft.
```

File: modules/ROOT/pages/intro-platform-hosting.adoc [hunk 4]
```diff
@@ -86,13 +87,21 @@ Not all Anypoint Platform components are supported in all cloud environments. Th
...
 +
 CloudHub also provides high availability, clustering and failover of your APIs and Mule applications and performs load balancing for them. See xref:runtime-manager::cloudhub.adoc[CloudHub] for more information.
 +
-CloudHub can only be managed only by version of the cloud control plane that exists in the same environment (US cloud, EU cloud, or MuleSoft Government Cloud).
+CloudHub can only be managed only by the version of the cloud control plane that exists in the same environment (US cloud, EU cloud, or MuleSoft Government Cloud).
```
Comment (vli63127) [inline modules/ROOT/pages/intro-platform-hosting.adoc]: this does not read grammatically pleasing. There are two "only's in one sentence.

Comment (sarathecoubian) [reply modules/ROOT/pages/intro-platform-hosting.adoc]:
```suggestion
CloudHub can be managed only by the version of the cloud control plane that exists in the same environment (US cloud, EU cloud, or MuleSoft Government Cloud).
```

File: modules/ROOT/pages/intro-platform-hosting.adoc [hunk 5]
```diff
@@ -86,13 +87,21 @@ Not all Anypoint Platform components are supported in all cloud environments. Th
 
 MuleSoft provides three different options for hosting the Anypoint Platform runtime plane:
 
+* CloudHub 2.0
++
+CloudHub 2.0 is MuleSoft's new default cloud-hosted environment for hosting the Mule runtime and related services. CloudHub 2.0 enables you to deploy an API or a Mule application into a container-based platform that's managed by MuleSoft. You can deploy to MuleSoft's shared spaces or your own configurable single-tenant private spaces for more control over what can interact with your applications.
++
+CloudHub 2.0 also provides high availability, ingress load balancers, AWS role-based access controls, and endpoint configuration. See xref:cloudhub-2::index.adoc[CloudHub 2.0] for more information.
++
+CloudHub 2.0 can only be managed only by the version of the cloud control plane that exists in the same environment (US cloud or EU cloud).
```
Comment (sarathecoubian) [inline modules/ROOT/pages/intro-platform-hosting.adoc]:
```suggestion
CloudHub 2.0 can be managed only by the version of the cloud control plane that exists in the same environment (US cloud or EU cloud).
```

File: modules/ROOT/pages/intro-platform-hosting.adoc [hunk 6]
```diff
@@ -86,13 +87,21 @@ Not all Anypoint Platform components are supported in all cloud environments. Th
 
 MuleSoft provides three different options for hosting the Anypoint Platform runtime plane:
 
+* CloudHub 2.0
++
+CloudHub 2.0 is MuleSoft's new default cloud-hosted environment for hosting the Mule runtime and related services. CloudHub 2.0 enables you to deploy an API or a Mule application into a container-based platform that's managed by MuleSoft. You can deploy to MuleSoft's shared spaces or your own configurable single-tenant private spaces for more control over what can interact with your applications.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/intro-platform-hosting.adoc]:
```suggestion
CloudHub 2.0 is MuleSoft's default cloud-hosted environment for hosting the Mule runtime and related services. CloudHub 2.0 enables you to deploy an API or a Mule application into a container-based platform that's managed by MuleSoft. You can deploy to MuleSoft's shared spaces or your own configurable single-tenant private spaces for more control over what can interact with your applications.
```

Comment (IsaacEldridge) [reply modules/ROOT/pages/intro-platform-hosting.adoc]: it's not always going to be "new"

Comment (IsaacEldridge) [reply modules/ROOT/pages/intro-platform-hosting.adoc]: "cloud-hosted" seems weird to me, but leave it if you want

File: modules/ROOT/pages/intro-platform-hosting.adoc [hunk 7]
```diff
@@ -86,13 +87,21 @@ Not all Anypoint Platform components are supported in all cloud environments. Th
...
++
+CloudHub 2.0 also provides high availability, ingress load balancers, AWS role-based access controls, and endpoint configuration. See xref:cloudhub-2::index.adoc[CloudHub 2.0] for more information.
++
+CloudHub 2.0 can be managed only by the version of the cloud control plane that exists in the same environment (US cloud or EU cloud).
+
 * CloudHub
 +
-CloudHub is MuleSoft's cloud-based environment for hosting the Mule runtime server and related services. CloudHub enables you to deploy an API or a Mule application on a platform that’s managed by MuleSoft.
+CloudHub is MuleSoft's original cloud-hosted environment for hosting the Mule runtime and related services. CloudHub enables you to deploy an API or a Mule application on a VM-based platform that’s managed by MuleSoft.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/intro-platform-hosting.adoc]:
```suggestion
CloudHub is MuleSoft's original cloud environment for hosting the Mule runtime and related services. CloudHub enables you to deploy an API or a Mule application on a VM-based platform that’s managed by MuleSoft.
```

Comment (IsaacEldridge) [reply modules/ROOT/pages/intro-platform-hosting.adoc]: I'm bristling at "original" and "new" but can think of a way around it. I def don't think we need to say "new" in docs, tho

**Review by IsaacEldridge — APPROVED** [review]

**Pull request merged**